Vagrantfile

```diff
@@ -5,6 +5,8 @@
 Vagrant.configure(2) do |config|
     config.vm.box = "centos/atomic-host"
     config.vm.hostname = "centosah-dev"
+    config.vm.define "vmcheck" do |vmcheck|
```

Comment:
This changes the name of the default VM to "vmcheck". I got tired of it being "default". :)